src/app/stores/TodoStore.js

```diff
@@ -19,12 +19,39 @@ class TodoStore {
       console.log("OPERATION")
       return operation(todoList);
     })
-
-    // each updated note array is stored in local storage
-    this.todoList.subscribe((update) => {
-      console.log('SAVE', update)
-      LocalStorage.save(key, update)
+    .flatMapLatest((todoList) => {
```

Comment:
So here is an issue I was having, this deals with the entire list. It seems like I'd persist remotely individual todo items, vs the entire array.